content/applications/inventory_and_mrp/inventory/management/products/usage.rst

```diff
@@ -1,72 +1,240 @@
+   2 :guilabel:`Done` products to `PACK0000006` and `PACK0000007` each.
+
+   .. image:: usage/put-in-pack-button.png
+      :align: center
+      :alt: Put in pack button to match the done amount matches the demand.
+
+Once all the products for each line have been entered, click :guilabel:`Confirm` to return to the
+:abbr:`DO (Delivery Order)`.
+
+The :guilabel:`Done` quantity on the :abbr:`DO (Delivery Order)` updates as products are selected
+for each package. When the :guilabel:`Done` amount matches the :guilabel:`Demand` amount, click the
+:guilabel:`Validate` button to complete the delivery order.
+
+.. tip::
+   The :guilabel:`Put In Pack` button is intended to quickly package all products in the :abbr:`DO
+   (Delivery Order)` into one new package. The option to rename the package, or modify its contents,
+   are available in the :guilabel:`Detailed Operations` pop-up.
+
+Clicking on the :guilabel:`Packages` smart button in the upper-right of the delivery order reveals a
+separate page, showing all the packages used in that :abbr:`DO (Delivery Order)`.
+
+Packagings
+==========
+
+*Packaging* is product specific, and refers to a disposable container that holds several units of a
+specific product. Unlike packages, packagings cannot be reusable, and each specific packaging must
+be defined on the individual product form.
+
+For example, different packages for cans of soda can be configured as a 6-pack, a 12-pack, or a case
+of 36. Each flavor of soda would need a `6`, `12`, and `36` can packaging configured on the
+individual product since packagings are product specific, not generic.
+
+Set up packagings
+-----------------
+
+To use packagings, ensure the :guilabel:`Product Packagings` feature is enabled. To do that,
+navigate to :menuselection:`Inventory app --> Configuration --> Settings`. Then, under the
+:guilabel:`Products` heading, check the box next to :guilabel:`Product Packagings`, and click
+:guilabel:`Save`.
+
+In Odoo, product packagings are used on sales/purchase orders and inventory transfers.
+
+Next, to create packagings, navigate to :menuselection:`Inventory app --> Products --> Products` and
+click on the desired product, then click :guilabel:`Edit` on the product page.
+
+Under the :guilabel:`Inventory` tab, scroll down to the :guilabel:`Packaging` section, and click
+:guilabel:`Add a line`. Then, a pop-up window appears, in which the following information for each
+packaging should be entered.
+
+- :guilabel:`Packaging`: Name of packaging that appears on sales/purchase orders as a packaging
+  option for the product.
+- :guilabel:`Contained quantity`: Amount of product in the packaging.
+- :guilabel:`Barcode`: Identifier used with the *Barcode* app to trace the packaging of a product
+  during stock moves or pickings. Leave blank if not in use.
+- :guilabel:`Company`: Indicates the packaging is only available at the selected company. Leave
+  blank to make the packaging available across all companies.
```

Comment:
Optional.
This is correct according to some style guides, however I am generally prefer Chicago-style for lists, which does not capitalize the first word after a colon.
Up to you though 😉

Reply:
*Barcode* on line 181 might be a good opportunity to link to the Barcode product page or Barcode documentation.